resume-buddy/src/main/webapp/script.js

```diff
@@ -0,0 +1,10 @@
+function blobUpload() {
+	fetch('/blobstore-upload-url')
+		.then((response) => {
+			return response.text();
+		})
+		.then((imageUploadUrl) => {
```

Comment:
`imageUploadUrl` should be changed with new name since it's not image upload url anymore